app/controllers/api/account.php

```diff
@@ -1060,7 +1062,7 @@
             throw new Exception('No valid session found', 404, Exception::USER_SESSION_NOT_FOUND);
         }
 
-        $jwt = new JWT(App::getEnv('_APP_OPENSSL_KEY_V1'), 'HS256', 900, 10); // Instantiate with key, algo, maxAge and leeway.
+        $jwt = new JWT($project->getId() == 'console' ? App::getEnv('_APP_OPENSSL_KEY_V1') : $project->getAttribute('jwtSecret'), 'HS256', 900, 10); // Instantiate with key, algo, maxAge and leeway.
```

Comment:
Can we set/cast these attributes (jwtSecret, databaseSecrets) globally on the `console` project object? Might be easier than forcing dev to remember adding this condition.

Reply:
may be adding a JWT resource will be a good idea, I'll try that.